core/iwasm/fast-jit/fe/jit_emit_memory.c

```diff
@@ -243,6 +269,11 @@ jit_compile_op_i32_load(JitCompContext *cc, uint32 align, uint32 offset,
         }
     }
 
+#if WASM_ENABLE_SHARED_MEMORY != 0
+    if (atomic && load_insn)
+        set_load_or_store_atomic(load_insn);
```

Comment:
Compiler reports `load_insn is set but not used` warning when shared memory isn't enabled. Had better add
```C
#else
    (void)load_insn;
```
to clear the warning.